packages/core/pubspec.yaml

```diff
@@ -9,6 +9,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
+  intl: ^0.17.0
```

Comment:
`intl` package should not be part of this PR, but to keep the overhead of managing this low, we can keep it in.

Reply:
I'm using it in the `src/widgets/tweet/metric_text.dart` widget to format the numbers with `NumberFormat` (e.g. `10000` => `10k`)

Should we not depend on it?

Reply:
Oh didn't realize that and confused it with localization 😀